src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerAwaitingApplicant.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.divorce.caseworker.event;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.api.CCDConfig;
+import uk.gov.hmcts.ccd.sdk.api.ConfigBuilder;
+import uk.gov.hmcts.divorce.common.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+import uk.gov.hmcts.divorce.divorcecase.model.UserRole;
+
+import static java.util.EnumSet.allOf;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingDocuments;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.CASE_WORKER;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.LEGAL_ADVISOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SOLICITOR;
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.SUPER_USER;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.CREATE_READ_UPDATE;
+import static uk.gov.hmcts.divorce.divorcecase.model.access.Permissions.READ;
+
+@Component
+@Slf4j
+public class CaseworkerAwaitingApplicant implements CCDConfig<CaseData, State, UserRole> {
+    public static final String CASEWORKER_AWAITING_APPLICANT = "caseworker-awaiting-applicant";
+
+    @Override
+    public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuilder) {
+        new PageBuilder(configBuilder
+            .event(CASEWORKER_AWAITING_APPLICANT)
+            .forStateTransition(allOf(State.class), AwaitingDocuments)
+            .name("Awaiting Applicant")
+            .description("Awaiting Applicant")
+            .explicitGrants()
```

Comment:
Is explicitGrants correct here?

Reply:
good catch will remove as it is not needed anymore